projects/desktop/src/components/RegisterScreen/index.tsx

```diff
@@ -1,29 +1,103 @@
-import React from 'react'
-import {useTranslation, UseTranslationResponse} from 'react-i18next'
-import {SingleFormContainer} from '../common/Containers/SingleFormContainer'
-import { BigTitle } from '../common/SingleForm'
-import {Box, Logo, RegisterLanguageChooser} from './styles'
+import React, { useReducer } from 'react'
+import { useTranslation, UseTranslationResponse } from 'react-i18next'
+import { SingleFormContainer } from '../common/Containers/SingleFormContainer'
+import {  BigTitle } from '../common/SingleForm'
+import { Box, Logo, RegisterLanguageChooser } from './styles'
 import logo from '../../assets/img/logo-small.png'
-import { Form as FinalForm } from 'react-final-form'
-import {Form} from './form'
+import { Form as FinalForm, AnyObject } from 'react-final-form'
+import { Form } from './form'
+import axios, { AxiosResponse } from 'axios'
+import { serverURL } from '../../constants'
+import _ from 'lodash'
+
```

Comment:
By the way, for all the imports, I think it is better to group the external libraries and then a blank line and then all the internal imports